apps/web/src/components/milestone-logs/recruitment/OfferAcceptedBanner.tsx

```diff
@@ -1,8 +1,37 @@
-interface OfferBannerProps {
-  className?: string;
-}
-export const OfferAcceptedBanner: React.FC<OfferBannerProps> = ({ children, className }) => {
+import { useApplicantContext } from 'src/components/applicant/ApplicantContext';
+import hiredIndIcon from '@assets/img/hired_indicator.svg';
+import hiredIndCheckmark from '@assets/img/hired_checkmark_banner.svg';
+import { isHired } from '@ien/common';
+
+export const OfferAcceptedBanner: React.FC = () => {
+  const { applicant, offerWithDiffHa } = useApplicantContext();
+
   const defaultStyle = `flex items-center h-14 mb-5 font-bold rounded`;
 
-  return <div className={`${defaultStyle} ${className}`}>{children}</div>;
+  // check for an accepted offer within the same HA as current user
+  const acceptedJobOfferWithSameHa = (): boolean | undefined => {
+    return applicant.jobs?.some(s => s.status_audit?.find(a => isHired(a.status.id)));
+  };
+
+  // show if an offer was accepted within a different HA than logged in user
+  if (offerWithDiffHa) {
```

Comment:
If possible, we'd better to have one component structure of html elements. It means we return one structure with dynamic properties.
With your approach, it is hard to understand the differences of these branches.